config/install/standard/kuadrant-version.yaml

```diff
@@ -3,4 +3,4 @@ kind: CatalogSource
 metadata:
   name: kuadrant-operator-catalog
 spec:
-  image: quay.io/kuadrant/kuadrant-operator-catalog:latest #change this to the version you want to install
+  image: quay.io/kuadrant/kuadrant-operator-catalog:release-v1.0.0 # change this to the version you want to install
```

Comment:
thinking about it, makes sense as `latest` on main. One less thing to worry about at release time.